documentation/asciidoc/computers/configuration/uart.adoc

```diff
@@ -116,12 +146,26 @@ Linux devices on Raspberry Pi OS:
 
 | `/dev/serial1`
 | secondary UART
+
+| `/dev/ttyAMA10`
+| Raspberry Pi 5 Debug UART
 |===
 
-NOTE: `/dev/serial0` and `/dev/serial1` are symbolic links which point to either `/dev/ttyS0` or `/dev/ttyAMA0`.
+`/dev/serial0` and `/dev/serial1` are symbolic links which point to either `/dev/ttyS0` or `/dev/ttyAMA0`.
+
+On the Raspberry Pi 5, `/dev/serial0` is a symbolic link that points to `/dev/ttyAMA10`. Due to changes in Bookworm, `/dev/serial1` does not exist by default on the Raspberry Pi 5. You can re-enable `serial1` by setting the following values in `config.txt`:
```

Comment:
Sorry, I just noticed the second "on the Pi 5" here, which shouldn't be there - you won't see /dev/serial1 on any Pi when the kernel is managing Bluetooth. It's probably worth at least splitting the paragraph after the first sentence, but the second sentence could be moved to make it more prominent since it in some part invalidates/obviates some of what has gone before. I'm not going to push for that - just dropping the 2nd "on the Pi 5" is enough to make it correct.

Reply:
```suggestion
On the Raspberry Pi 5, `/dev/serial0` is a symbolic link that points to `/dev/ttyAMA10`.

Due to changes in Bookworm, `/dev/serial1` does not exist by default. You can re-enable `serial1` by setting the following values in `config.txt`:
```